app/views/users/invitations/edit.html.erb

```diff
@@ -31,6 +31,17 @@
     <span><small><%= t 'users.registrations.new.team_name_help' %></small></span>
   </div>
 
+  <% if Rails.configuration.x.enable_recaptcha %>
```

Comment:
You can use `<%= recaptcha_input_tag %>` that @mlorb implemented (see [form_tag_helper.rb](https://github.com/biosistemika/scinote-web/blob/simplify-signup/app/helpers/form_tag_helper.rb)) here.